bubuku/daemon.py

```diff
@@ -48,7 +48,10 @@ def apply_features(api_port, features: dict, controller: Controller, buku_proxy:
                     host=env_provider.get_ip(),
                     port=port
                 ))
+            kafka_properties.set_property('advertised.listeners', ",".join(new_listeners))
+            new_listeners.append("PLAINTEXT://localhost:9092")
```

Comment:
Well, it will probably work, but I would do it a bit more resilient:
1. use `127.0.0.1` instead of `localhost`
2. try to avoid situation where duplicate records are added

Regarding second part: 
imagine that in old configuration it is written 
```
listeners=PLAINTEXT://localhost:9092,SSL://localhost:9093
```
I would expect, that that is what I would get after turning feature `use_ip_address`:
```
listeners=PLAINTEXT://127.0.0.1:9092,SSL://127.0.0.1:9093,PLAINTEXT://some_ip:9092,SSL://some_ip:9093
advertized.listeners=PLAINTEXT://some_ip:9092,SSL://some_ip:9093
```
It doesn't look like it will work with the approach you took